runtime/t0rn-parachain/src/parachain_config.rs

```diff
@@ -107,7 +107,7 @@ impl pallet_session::Config for Runtime {
 impl pallet_aura::Config for Runtime {
     type AllowMultipleBlocksPerSlot = ConstBool<false>;
     type AuthorityId = AuraId;
-    type DisabledValidators = ();
+    type DisabledValidators = Session;
```

Comment:
> A way to check whether a given validator is disabled and should not be authoring blocks. Blocks authored by a disabled validator will lead to a panic as part of this module’s initialization.

https://docs.rs/pallet-aura/latest/pallet_aura/pallet/trait.Config.html#associatedtype.DisabledValidators

So what will happen when we will switch it to `Session`?

Reply:
Session exposes interface to check disabled validators. currently however the list of disabledValidators is empty on t0rn so we'd need to pin slashing rules that would ask Session::disableIndex